vspec2x.py

```diff
@@ -53,6 +53,8 @@ def from_string(s):
 def main(arguments):
     parser.add_argument('-I', '--include-dir', action='append',  metavar='dir', type=str,  default=[],
                         help='Add include directory to search for included vspec files.')
+    parser.add_argument('-e', '--extended-attributes', type=str,  default="",
+                        help='Whitelisted extended attributes as comma sperated list. Note, that not all exporters will support (all) extended attributes.')
     parser.add_argument('-s', '--strict', action='store_true',
                         help='Use strict checking: Terminate when anything not covered or not recommended by the core VSS specs is found.')
```

Comment:
In some way we should rephrase this as the term "core VSS specs" is not true any longer as we consider "extensions". 

Maybe something like:

"Terminate when anything not covered or not recommended by VSS language or extensions is found""